shell/platform/linux/fl_accessible_node.cc

```diff
@@ -46,7 +47,6 @@ static ActionData action_mapping[] = {
      "MoveCursorForwardByCharacter"},
     {kFlutterSemanticsActionMoveCursorBackwardByCharacter,
      "MoveCursorBackwardByCharacter"},
-    {kFlutterSemanticsActionSetSelection, "SetSelection"},
```

Comment:
Why is this removed?

Reply:
The SetSelection action cannot be executed "standalone" because it requires the target selection as an [argument](https://github.com/flutter/flutter/blob/712860d33b00bf3acf02118512901a6261ad5219/packages/flutter/lib/src/semantics/semantics.dart#L3608-L3616). Performing the action would result to:
```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: 'package:flutter/src/semantics/semantics.dart': Failed assertion: line 3609 pos 14: 'args != null && args is Map': is not true.
#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:51:61)
#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:40:5)
#2      SemanticsConfiguration.onSetSelection=.<anonymous closure> (package:flutter/src/semantics/semantics.dart:3609:14)
#3      SemanticsOwner.performAction (package:flutter/src/semantics/semantics.dart:3131:14)
#4      RendererBinding._handleSemanticsAction (package:flutter/src/rendering/binding.dart:362:36)
#5      _invoke3 (dart:ui/hooks.dart:207:13)
#6      PlatformDispatcher._dispatchSemanticsAction (dart:ui/platform_dispatcher.dart:1030:5)
#7      _dispatchSemanticsAction (dart:ui/hooks.dart:99:31)
```

Removing the action seemed reasonable given that `FlAccessibleTextField` implements `AtkText::set_selection` and [AtkAction](https://gnome.pages.gitlab.gnome.org/atk/AtkAction.html) docs say:
> Exceptions: when the user interaction is already covered by another appropriate interface such as [AtkEditableText](https://gnome.pages.gitlab.gnome.org/atk/AtkEditableText.html) (insert/delete text, etc.) or [AtkValue](https://gnome.pages.gitlab.gnome.org/atk/AtkValue.html) (set value) then these actions should not be exposed by [AtkAction](https://gnome.pages.gitlab.gnome.org/atk/AtkAction.html) as well.